packages/widget-chat/src/components/activity-item/index.js

```diff
@@ -1,21 +1,35 @@
 import React, {PropTypes} from 'react';
+import classNames from 'classnames';
+import Avatar from '../avatar';
+
+import styles from './styles.css';
 
 export default function ActivityItem(props) {
   const {
     content,
-    id
+    name,
+    timestamp
   } = props;
 
   return (
-    <div>
-      <li>
-        {id} - {content}
-      </li>
+    <div className={classNames(`activity-item`, styles.activityItem)}>
+      <div className={classNames(`avatar-wrapper`, styles.avatarWrapper)}>
+        <Avatar name={name} />
+      </div>
+      <div className={classNames(styles.content)}>
```

Comment:
Good question. This is more of an oversight on my part than anything else. But the larger discussion point is, do we want to make everything overridable? Or just some things. At this point I think it's fine to make all of the things overridable.